server/src/main/java/org/activityinfo/ui/client/page/entry/location/LocationSearchPresenter.java

```diff
@@ -94,10 +94,10 @@ private void numberLocations(List<LocationDTO> locations) {
         int number = 0;
         for (LocationDTO location : locations) {
             if (location.hasCoordinates()) {
-                location.setMarker(String.valueOf((char) ('A' + number)));
                 number++;
+                location.setMarker(number.toString());
```

Comment:
Yes I will set up the env so I can run tests and look into the marker issue, lets keep this PR open whilst I work on the fix. So two digits is out? ie up to 99?